sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -1,5 +1,9 @@
 {
+  "title": "Institution Welcome Page",
+  "display": "form",
   "type": "form",
+  "name": "institutionwelcomepage",
```

Comment:
Why is the name and path showing as modified here?

Reply:
it's not modified, the form was in the old format, I updated it to the new format. that's why you are seeing it as modified, 
![image](https://user-images.githubusercontent.com/77353155/186249829-47013598-1e1e-46df-8aae-09923664b684.png)

Reply:
Yeah it looks like a formatting change which came in. I was assuming that @andrewsignori-aot modified all the form definitions to format with json formatter lately. Hence I was confused how come formatting are happening again.